### PR TITLE
Awesome Books - ES6 Compatible

### PR DESCRIPTION
**Pull request by ehmaddd**
This project is the modification of the previous project which was created to view, add and delete books from the webpage divided into sections. In this update, the whole project is divided into different modules and every module is included in the main JS File which gives and takes data to/from the module file. The module is basically a chunk of code separated to avoid the mess in the main file. Moreover, Luxon is used to getting live date/times on the webpage.

### CODE REVIEW

**Review by ouasamine — CHANGES_REQUESTED** [review]
Hi @ehmaddd 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Nice work using the ES6 syntax correctly 👏🏽 👏🏽 
- Your code is well organised, good job ✔️ ✔️ 
- Good project's documentation 👍🏽 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/book-auth.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  } else {
+    book = {
+      title: titleText,
+      author: authorText,
+    };
+  }
+  return book;
+};
+
+export default bookAuth;
```
[the remark this answers is not in the document]
Comment (ehmaddd) [reply modules/book-auth.js:26]: Thanks for the complement. I have made changes as per instructions. Double export is removed.

File: modules/data-display.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      bookList.appendChild(li);
+    }
+  } else {
+    const h3 = document.createElement('h3');
+    h3.innerHTML = 'No record found';
+    bookList.appendChild(h3);
+  }
+};
+
+export default dataDisplay;
```
[the remark this answers is not in the document]
Comment (ehmaddd) [reply modules/data-display.js:27]: Changes have been made as instructed.

File: modules/nav-dynamics.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        nav[2].classList.add('active-link');
+        bookCatalog.style.display = 'none';
+        addBookForm.style.display = 'none';
+        contactInfo.style.display = 'block';
+      }
+    });
+  }
+};
+
+export default navBar;
```
[the remark this answers is not in the document]
Comment (ehmaddd) [reply modules/nav-dynamics.js:34]: Changes have been made as instructed.

File: modules/remove-list.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+export const removeList = () => {
+  const bookList = document.querySelector('.book-list');
+  while (bookList.firstChild) {
+    bookList.removeChild(bookList.firstChild);
+  }
+};
+
+export default removeList;
```
[the remark this answers is not in the document]
Comment (ehmaddd) [reply modules/remove-list.js:8]: Changes have been made as instructed.

File: modules/show-time.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  const today = DateTime.now();
+  const date = `${monthNames[(today.month + 1)]} ${today.day} ${today.year}`;
+  const time = `${today.hour}:${today.minute}:${today.second}`;
+  const dateTime = `<p>${date}</p><h3>${time}</h3>`;
+
+  timeDiv.innerHTML = dateTime;
+  setTimeout(showTime, 50);
+};
+
+export default showTime;
```
[the remark this answers is not in the document]
Comment (ehmaddd) [reply modules/show-time.js:15]: Changes have been made as instructed.

File: modules/window-load.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+  nav[0].classList.add('active-link');
+  addBookForm.classList.add('hide');
+  contactInfo.classList.add('hide');
+  bookCatalog.style.display = 'block';
+  addBookForm.style.display = 'none';
+  contactInfo.style.display = 'none';
+};
+
+export default windowLoad;
```
[the remark this answers is not in the document]
Comment (ehmaddd) [reply modules/window-load.js:15]: Changes have been made as instructed.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+        <img src="./images/ab.png">
+        <nav>
+            <ul>
+                <li><a id="showList">List</a></li>
+                <li><a id="showForm">Add new</a></li>
+                <li><a id="showContact">Contact</a></li>
+            </ul>
+        </nav>
+        <div id="time-div"></div>
+    </header>
```
[the remark this answers is not in the document]
Comment (ehmaddd) [reply index.html:21]: Thanks for the kind patronization. I've changed the default space to 2 and also will change the default size of tab key in VS Code.

**Review by shella12 — APPROVED** [review]
Hi @ehmaddd , 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Linters are passing ✔️ 
- README is professional 👍 
- Great UI 👍 
- Implemented all requirements 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**